.gitlab-ci.yml

```diff
@@ -0,0 +1,56 @@
+stages:
+    - build
+    - deploy
+
+.maven-build:
+    image: maven:3.5.4-jdk-8
+    stage: build
+    variables:
+        ARTIFACT_PATH: "eva-server/target/eva-$ENVIRONMENT_NAME.war"
+    script:
+        - "mvn package --settings .gitlab.settings.xml -P $MAVEN_PROFILE"
+        - "cp eva-server/target/eva.war $ARTIFACT_PATH"
+    artifacts:
+        paths:
+            - "$ARTIFACT_PATH"
```

Comment:
I think we should add the 'expire' attribute now that we have it fresh in our minds, otherwise we will forget about it and end up with lots of artifacts that we don't want to keep. 2 weeks sounds like a reasonable expiration period to me.

Reply:
```suggestion
            - "$ARTIFACT_PATH"
        expire_in: 2 weeks
```